pkg/types/hashedrekord/v0.0.1/entry.go

```diff
@@ -160,11 +159,6 @@ func (v *V001Entry) validate() (pki.Signature, pki.PublicKey, error) {
 		return nil, nil, types.ValidationError(err)
 	}
 
-	_, isEd25519 := keyObj.CryptoPubKey().(ed25519.PublicKey)
```

Comment:
Just for context, when I wrote up https://github.com/sigstore/rekor/issues/1325, my intention was to support ed25519ph only for hashedrekord, and ed25519 for all other types. This might mean making some changes to the 'pkg/pki/x509/x509' library to support loading in a custom verifier like on https://github.com/sigstore/rekor/blob/70d6663e297068383f708155f4017110c6f46713/pkg/pki/x509/x509.go#L87

Reply:
What do you think about [this](https://github.com/sigstore/rekor/pull/1945/commits/2d93686760f8fe40ea552ca9895c2922bfcb7780#diff-1cad0eed4767a1ef5dbac388d3e45fc4fe1a3c4fb6187f572040b515d81660b8) way of customizing the verifier?